src/io/tree.cpp

```diff
@@ -310,6 +310,14 @@ std::string Tree::ToIfElse(int index, bool is_predict_leaf_index) {
   if (num_leaves_ == 1) {
     str_buf << "return 0";
   } else {
+    str_buf << "std::vector<uint32_t> cat_threshold = {";
```

Comment:
@guolinke should I use `constexpr`? and should I put it outside of function `PredictTree`?